### PR TITLE
Set speedUpDuration to 15 days in VeJoeStaking deploy

### PR DESCRIPTION
**Pull request by 0xnivek**
Noticed its currently set to 15 seconds rather than 15 days

### CODE REVIEW

**Review by jummy123 — APPROVED** [review]
LGTM, once merged I will deploy today. Do you think we need to add a function to update this variable? `setSpeedUpDuration(uint256 seconds)`?

**Comment by 0xnivek** [comment]
> LGTM, once merged I will deploy today. Do you think we need to add a function to update this variable? `setSpeedUpDuration(uint256 seconds)`?

Supporting dynamic `speedUpDuration` would require some additional logic changes and from speaking w/ murloc it seems like we're fine keeping it as static value

**Pull request merged**
